FirebaseDatabase.podspec

```diff
@@ -16,9 +16,16 @@ Simplify your iOS development, grow your user base, and monetize more effectivel
     :tag => 'CocoaPods-' + s.version.to_s
   }
   s.social_media_url = 'https://twitter.com/Firebase'
-  s.ios.deployment_target = '10.0'
-  s.osx.deployment_target = '10.12'
-  s.tvos.deployment_target = '10.0'
+
+  ios_deployment_target = '10.0'
+  osx_deployment_target = '10.12'
+  tvos_deployment_target = '10.0'
+  watchos_deployment_target = '7.0'
```

Comment:
Note: for lifecycle changes, watchOS 7 is preferred but the SwiftPM manifest currently states 6. We need to bump it at the next major change.

Reply:
Please open a Firebase 8 issue to track

Reply:
Done in https://github.com/firebase/firebase-ios-sdk/issues/7663. Alternatively I could mark up  the API with `@available(watchOS 7, *)` and that may be a better solution.  WDYT?